### ISSUE TITLE
installation script on Ubuntu 24.04 LTS fails

### ISSUE THREAD

**Issue by ajnadox** [chatwoot/chatwoot#9570]
### Describe the bug

setting up Ubuntu 24.04 LTS  and running installation fails at stage 6

➥ 4/9 Setting up Ruby
➥ 5/9 Setting up the database.
➥ 6/9 Installing Chatwoot. This takes a long while.

Some error has occured. Check '/var/log/chatwoot-setup.log' for details.
root@support:~#

and checking chatwoot-setup.log ends with:

ALTER ROLE
ALTER ROLE
UPDATE 1
DROP DATABASE
CREATE DATABASE
UPDATE 1
You are now connected to database "template1" as user "postgres".
VACUUM
Synchronizing state of redis-server.service with SysV service script with /usr/lib/systemd/systemd-sysv-install.
Executing: /usr/lib/systemd/systemd-sysv-install enable redis-server
Synchronizing state of postgresql.service with SysV service script with /usr/lib/systemd/systemd-sysv-install.
Executing: /usr/lib/systemd/systemd-sysv-install enable postgresql
This account is currently not available.
root@support:~# 

### To Reproduce

sudo -i
wget https://get.chatwoot.app/linux/install.sh
chmod +x install.sh
./install.sh --install

### Environment

Linux VM

### Cloud Provider

GCP

### Platform

None

### Operating system

Ubuntu 24.04 LTS

**Comment by vishnu-narayanan** [comment]
https://www.chatwoot.com/docs/self-hosted/deployment/requirements#supported-linux-distribution

**Comment by ajnadox** [comment]
Yes i know but 20.04 is 4 years old and we have a new LTS, im willing to help finding Compatibility issues, getting on it right now :)

**Comment by ajnadox** [comment]
yes i know, hade it running on 22.04, trashed that VM and should setup a new on 24.04 and this happend ;)

**Comment by ajnadox** [comment]
@vishnu-narayanan pull requests 9573 9572 and 9571
Installation script is now compatible with ubuntu 24.04 LTS i have it up and running.

**Comment by vishnu-narayanan** [comment]
@ajnadox Thank you for the PR's
